extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py

```diff
@@ -506,6 +506,7 @@ def to_dict(self):
         model['_c_dl'] = copy.deepcopy(self._c_dl)
         model['_c_lw'] = copy.deepcopy(self._c_lw)
         model['_c_l'] = copy.deepcopy(self._c_l)
+        model = self._make_json_serializable(model)
         return model
 
     def from_dict(self, model):
```

Comment:
Should this change be a part of this PR @seanlip ? . Right now, we are combining a lot of different fixes in this one PR. Should I make a separate issue for the above review?